core/src/main/scala/io/projectglow/transformers/LiftOverVariantsTransformer.scala

```diff
@@ -206,7 +215,9 @@ object LiftOverVariantsTransformer extends GlowLogging {
       inputVc: VariantContext,
       liftOver: LiftOver,
       minMatchRatio: Double,
-      refSeqMap: Map[String, ReferenceSequence]): (Option[VariantContext], Option[String]) = {
+      refSeqMap: Map[String, ReferenceSequence],
+      attributesToSwap: Seq[String],
```

Comment:
Maybe it would be more clear to call this `infoFieldsToSwap`? And `formatFieldsToSwap` or `genotypeFieldsToSwap`? I don't think it's helpful to use the htsjdk terminology as it's pretty bad.